reference/multigrid/pmis_kernels.cpp

```diff
@@ -32,7 +33,45 @@ void compute_strong_dep_row(std::shared_ptr<const DefaultExecutor> exec,
                             remove_complex<ValueType> strength_threshold,
                             IndexType* sparsity_rows)
 {
-    GKO_NOT_IMPLEMENTED;
+    using rc = remove_complex<ValueType>;
+
+    const auto nrow = csr->get_size()[0]; 
+    const auto row_ptrs = csr->get_const_row_ptrs();   
+    const auto col_idxs = csr->get_const_col_idxs();   
+    const auto vals     = csr->get_const_values();  
+
+    sparsity_rows[0] = IndexType{0};
+
+    for (auto i = 0; i < nrow; ++i) {
+        
+        sparsity_rows[i+1] = sparsity_rows[i];
+
+        const auto row_start = row_ptrs[i];
+        const auto row_end   = row_ptrs[i + 1];
+
+        for (auto j = row_start; j < row_end; ++j) {
+            const auto k = col_idxs[j];
+            if (k == i) continue; 
```

Comment:
```suggestion
            if (k == i) {continue;} 
```
It is from our format guildline.
we always use `{}` even when it only contains one line.